hack/deploy-image.sh

```diff
@@ -13,6 +13,29 @@ LOCAL_PORT=${LOCAL_PORT:-5000}
 
 tag=${tag:-"127.0.0.1:${registry_port}/$IMAGE_TAG"}
 
+if [ "${USE_IMAGE_STREAM:-false}" = true ] ; then
+    oc process \
+        -p CL_OP_GITHUB_URL=https://github.com/${CL_OP_GITHUB_REPO:-openshift}/cluster-logging-operator \
+        -p CL_OP_GITHUB_BRANCH=${CL_OP_GITHUB_BRANCH:-master} \
+        -f hack/image-stream-build-config.yaml | \
```

Comment:
This file appears to be missing.  Is it a template like? https://github.com/openshift/cluster-logging-operator/pull/88/files#diff-571fb67fc88fd2f824bfc43d0c3ecc79R1

Reply:
fixed